php/src/Google/Protobuf/Internal/Descriptor.php

```diff
@@ -162,6 +163,16 @@ public function getLegacyClass()
         return $this->legacy_klass;
     }
 
+    public function setPreviousClass($klass)
```

Comment:
I lean a bit towards this being too verbose, but what about `setPreviouslyReservedClass`? If that's too much, maybe we can just add a note to the docblock about the intent, otherwise the intent isn't too clear.

Reply:
Done in https://github.com/protocolbuffers/protobuf/pull/10041